pages/ident.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+  <div>
+    <div class="h-screen bg-dark flex justify-center items-center">
```

Comment:
Why is it usefull to use `h-screen`?

Reply:
What would be your recommendation?

Reply:
First i need to understand why there is `100vh` used?

Reply:
Because we want to fill in the whole site with the dark background-color.

Reply:
ok.